### PR TITLE
Phase2-hgx359J Add new code for V19 version of HGCal geometry

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add new code for V19 version of HGCal geometry

#### PR validation:

Test demands a new wafer type for 200-micron HD wafers. We are working on that

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by srimanob** [comment]
+Upgrade

Failure in D110 DD4hep workflow is know issue.

File: Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedFineCassette.cc [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #include <unordered_set>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```
Comment (civanch) [inline Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedFineCassette.cc:34]: @bsunanda , please, comment this out.

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedFineCassette.cc (hunk 1 above)
Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalMixRotatedFineCassette.cc]: done

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Could you please approve this PR again?

**Comment by srimanob** [comment]
+Upgrade

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**